resources/assets/javascripts/admin.js

```diff
@@ -12,13 +12,15 @@ jQuery(function() {
   var $syncUrlButton = $('input[name=sync-url]')
   var $syncUrl = $('input[name=url]')
   var $status = $('.import-status')
+  var $speedtest = $('.speedtest-status')
   var ajaxUrl = $('.ajax-url').html()
 
   $syncCategoryButton.click(onSyncCategoryClick)
   $syncUrlButton.click(onSyncUrlClick)
 
   listSections()
   getCurrentStatus()
+  runSpeedtest()
```

Comment:
This looks to me like a process/network heavy thing to do every time the admin user wants to import a URL. Can we put this runSpeedTest behind a user action, such as a button Run speed test?